charts/cloud-connector/values.yaml

```diff
@@ -127,6 +127,8 @@ ingestors: []
 scanners: []
 
 bruteForceDetection:
+  # Enable Brute Force detection
```

Comment:
since this is the only available documentation right now for the user, could we briefly explain what it does?